fet.sh

```diff
@@ -100,7 +100,7 @@ read -r model < /sys/devices/virtual/dmi/id/product_name
 [ $# -gt 0 ] && pkgs=$#
 
 print() {
-	[ "$2" ] && printf '\033[3%sm%6s\033[0m ~ %s\n' "${accent:-4}" "$1" "$2"
+	[ "$2" ] && printf '\033[3%sm%6s\033[0m '"${seperator:-~}"' %s\n' "${accent:-4}" "$1" "$2"
```

Comment:
Don't use variables in the printf sequence, if you want it to accept escape codes use `%b` instead of `%s`.

Reply:
Also it's misspelt, it should be sep**a**rator instead of seperator